### PR TITLE
Added deadman to apps

### PR DESCRIPTION
**Pull request by Mikunj**

### CODE REVIEW

File: app/frontend/app/routes/apps/index.js [hunk 1]
```diff
@@ -105,6 +105,20 @@ export default Ember.Route.extend({
           { "name": "Discuss", "link": "https://forums.hummingbird.me/t/windows-hummingbird-updater-scrobbler-by-dennis/17333" },
           { "name": "Download", "link": "https://github.com/Desuvader/Hummingbird-Updater/releases/latest" },
         ]
+      },
+      {
+        "name": "Deadman",
+        "author": [
+          { "name": "Jigglyslime", "link": "https://www.youtube.com/watch?v=dQw4w9WgXcQ" },
```
Comment (NuckChorris) [inline app/frontend/app/routes/apps/index.js]: pls jiggly no rickroll

try a newer meme

Comment (Mikunj) [inline app/frontend/app/routes/apps/index.js]: :(

**Comment by cybrox** [comment]
+1 for vik's profile, cuz sloth.

File: app/frontend/app/routes/apps/index.js [hunk 2]
```diff
@@ -105,6 +105,20 @@ export default Ember.Route.extend({
...
+        "name": "Deadman",
+        "author": [
+          { "name": "Jigglyslime", "link": "http://hummingbird.me/users/Jigglyslime" },
+        ],
+        "form": "Mobile",
+        "platform": "iOS", 
+        "screenshots": [sb+"deadman/profile.jpeg", sb+"deadman/anime.jpeg"],
+        "desc": "It's like Hummingbird, but iOS.  And native.  Makes love to your face. Enjoy.",
+        "links": [
+          { "name": "Discuss", "link": "http://forums.hummingbird.me/t/ios-deadman/20315" },
```
Comment (fractalemagic) [inline app/frontend/app/routes/apps/index.js]: Should probably use HTTPS here

File: app/frontend/app/routes/apps/index.js [hunk 3]
```diff
@@ -105,6 +105,20 @@ export default Ember.Route.extend({
           { "name": "Discuss", "link": "https://forums.hummingbird.me/t/windows-hummingbird-updater-scrobbler-by-dennis/17333" },
           { "name": "Download", "link": "https://github.com/Desuvader/Hummingbird-Updater/releases/latest" },
         ]
+      },
+      {
+        "name": "Deadman",
+        "author": [
+          { "name": "Jigglyslime", "link": "http://hummingbird.me/users/Jigglyslime" },
```
Comment (NuckChorris) [inline app/frontend/app/routes/apps/index.js]: Should also use HTTPS here too

**Comment by JoshFabian** [comment]
Looks good to me!

**Comment by vikhyat** [comment]
:+1:

File: app/frontend/app/routes/apps/index.js [hunk 4]
```diff
@@ -105,6 +105,20 @@ export default Ember.Route.extend({
...
+      },
+      {
+        "name": "Deadman",
+        "author": [
+          { "name": "Jigglyslime", "link": "https://hummingbird.me/users/Jigglyslime" },
+        ],
+        "form": "Mobile",
+        "platform": "iOS", 
+        "screenshots": [sb+"deadman/deadman-1.jpg", sb+"deadman/deadman-2.jpg"],
+        "desc": "It's like Hummingbird, but iOS.  And native.  Makes love to your face. Enjoy.",
```
Comment (Wopian) [inline app/frontend/app/routes/apps/index.js]: What's with all the double spaces in the `desc`?

Comment (Mikunj) [inline app/frontend/app/routes/apps/index.js]: Nuck broke it

**Pull request merged**